src/server/pps/cmds/cmds.go

```diff
@@ -301,6 +301,22 @@ Examples:
 		}),
 	}
 
+	killJob := &cobra.Command{
+		Use:   "kill-job kill-id",
+		Short: "Kill a job.",
+		Long:  "Kill a job.  The job will be stopped immediately.",
```

Comment:
Not sure if that belongs to a help message.  I think users will be able to grasp what `kill` does regardless.